code/API_definitions/qod-booking.yaml

```diff
@@ -0,0 +1,1303 @@
+        applicationServerPorts:
+          description: A list of single ports or port ranges on the application server
+          allOf:
+            - $ref: "#/components/schemas/PortsSpec"
+        sink:
+          type: string
+          format: url
+          description: The address to which events shall be delivered, using the HTTP protocol.
+          example: "https://endpoint.example.com/sink"
+        sinkCredential:
+          $ref: "#/components/schemas/SinkCredential"
+      required:
+        - device
+        - qosProfile
+
+    BookingInfo:
+      description: |
+        Booking related information returned in responses.
+        Optional device object only to be returned if provided in createBooking. If more than one type of device identifier was provided, only one identifier will be returned (at implementation choice and with the original value provided in createBooking).
+        Please note that IP addresses of devices can change and get reused, so the original values may no longer identify the same device. They identified the device at the time of QoD Booking creation.
+      allOf:
+        - $ref: "#/components/schemas/BaseBookingInfo"
+        - type: object
+          properties:
+            statrTime:
+              description: Date and time when the platform of the API Provider switches the QoS flow with a requested QoS Profile. Format must follow RFC 3339 and must indicate time zone (UTC or local).
+              type: string
+              format: date-time
+              example: "2024-06-01T12:00:00Z"
+            duration:
+              description: |
+                Session duration in seconds. Implementations can grant the requested session duration or set a different duration from startTime, based on network policies or conditions.
+                - When `qosStatus` is "REQUESTED" or "SCHEDULED", the value is the duration to be scheduled, granted by the implementation.
+                - When `qosStatus` is "AVAILABLE", the value is the overall duration since `startedAt.
+                - When `qosStatus` is "UNAVAILABLE", the value is the overall effective duration since `startedAt` until the session was terminated.
+              type: integer
+              format: int32
+              minimum: 1
+              example: 3600
+            serviceArea:
+              $ref: "#/components/schemas/Area"
+            BookingId:
+              $ref: "#/components/schemas/BookingId"              
+            startedAt:
+              description: Date and time when the Booking became "AVAILABLE". Not to be returned when `status` is "REQUESTED" or "SCHEDULED". Format must follow RFC 3339 and must indicate time zone (UTC or local).
+              type: string
+              format: date-time
+              example: "2024-06-01T12:00:00Z"
+            status:
+              $ref: "#/components/schemas/Status"
+            statusInfo:
+              $ref: "#/components/schemas/StatusInfo"
+          required:
```

Comment:
In the response, other properties must be required: startTime, duration . startedAt has no sense in the response to CreateBooking but it is relevant for GET.